frame/vesting/README.md

```diff
@@ -15,6 +15,8 @@ equivalent to the amount remaining to be vested. This is done through a dispatch
 either `vest` (in typical case where the sender is calling on their own behalf) or `vest_other`
 in case the sender is calling on another account's behalf.
 
+A user can have multiple vesting schedules, which are capped by an upper bound.
```

Comment:
FYI I think these files are auto-generated from the CI form the tip level pallet doc, no need to write to them